phpstan.neon.dist

```diff
@@ -1,6 +1,6 @@
 includes:
     - phpstan-baseline.neon
-    - %ShopwareRoot%/vendor/phpstan/phpstan/conf/bleedingEdge.neon
+    - phar://phpstan.phar/conf/bleedingEdge.neon
     - %ShopwareRoot%/src/Core/DevOps/StaticAnalyze/PHPStan/extension.neon
     - %ShopwareRoot%/src/Core/DevOps/StaticAnalyze/PHPStan/rules.neon
```

Comment:
maybe you can deprecate with `tag:v10.0.0` so we switch to it asap

Reply:
it is not that important :slightly_smiling_face: I only found out today, that the bleeding edge config of PHPStan could be referenced like that. 

We need to put PHPStan into a `vendor-bin` directory in platform soon, so that's why it needs a change in every plugin. but with the phar reference it should also work